src/Swarm/TUI/View/CellDisplay.hs

```diff
@@ -58,9 +79,47 @@ displayLoc showRobots g coords =
       then displayRobotCell g coords
       else []
 
-data HideEntity = HideAllEntities | HideNoEntity | HideEntityUnknownTo Robot
+-- | Add random "static" based on the distance to the robot being
+--   @view@ed.
+addStatic :: GameState -> W.Coords -> Display -> Display
+addStatic g coords d = maybe d displayStatic (getStatic g coords)
 
-hidingMode :: GameState -> HideEntity
-hidingMode g
-  | g ^. creativeMode = HideNoEntity
-  | otherwise = maybe HideAllEntities HideEntityUnknownTo $ focusedRobot g
+-- | Draw black or white static.
+displayStatic :: Bool -> Display
+displayStatic = bool mempty (terrainMap M.! IceT)
```

Comment:
Cool effect.  It really ramps up the immersion into the "remote exploration" concept.

Take or leave this idea: once the `Far` range is reached, overlay a `[connection lost]` message in red color.